azurelinuxagent/common/logcollector.py

```diff
@@ -73,141 +102,50 @@ def _read_manifest_file(self):
 
     @staticmethod
     def _log_to_results_file(entry):
-        if type(entry) == ustr or type(entry) == str:
-            append_file(OUTPUT_RESULTS_FILE_PATH, entry + "\n")
+        if isinstance(entry, list):
+            append_items_to_file(_OUTPUT_RESULTS_FILE_PATH, entry)
         else:
-            append_items_to_file(OUTPUT_RESULTS_FILE_PATH, entry)
+            append_file(_OUTPUT_RESULTS_FILE_PATH, entry + "\n")
```

Comment:
I noticed that you're not logging the timestamp in the log file, it would be awesome if you could add that too. Thanks! :)

Reply:
Thanks for catching that, definitely agree it's useful. Added.